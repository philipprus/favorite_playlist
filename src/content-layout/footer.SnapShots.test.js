import React from "react";
import Footer from "./footer";
import renderer from "react-test-renderer";

it("render footer correctly", () => {
    const tree = renderer.create(<Footer/>).toJSON();

    expect(tree).toMatchSnapshot();

})