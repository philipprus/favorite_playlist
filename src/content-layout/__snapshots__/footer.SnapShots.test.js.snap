// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render footer correctly 1`] = `
<div
  className="MuiContainer-root makeStyles-footer-1 MuiContainer-maxWidthLg"
>
  <p>
    Powered API by
  </p>
  <img
    alt="Logo MusicMatch"
    src={[Function]}
  />
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="MuiContainer-root makeStyles-footer-1 MuiContainer-maxWidthLg"
>
  <p>
    Powered API by
  </p>
  <img
    alt="Logo MusicMatch"
    src={[Function]}
  />
</div>
`;
